guides/front_end/react_quick_start.mdx

```diff
@@ -0,0 +1,130 @@
+
+```bash
+npm install react-instantsearch @meilisearch/instant-meilisearch instantsearch.css
+
+# or
+yarn add react-instantsearch @meilisearch/instant-meilisearch instantsearch.css
+
+# or
+pnpm add react-instantsearch @meilisearch/instant-meilisearch instantsearch.css
+```
+
+## 3. Initialize the search client
+
+Use the following URL and API key to connect to a Meilisearch instance which contains data from Steam video games.
+
+```jsx
+import React from 'react';
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
+
+const { searchClient } = instantMeiliSearch(
+  'https://ms-adf78ae33284-106.lon.meilisearch.io',
+  'a63da4928426f12639e19d62886f621130f3fa9ff3c7534c5d179f0f51c4f303'
+);
+```
+
+## 4. Add InstantSearch
+
+`<InstantSearch>` is the mandatory wrapper component for our instant search. It takes two props: the `searchClient` and the [index name](/learn/core_concepts/indexes#index-uid).
+
+```jsx
+import React from 'react';
+import { InstantSearch } from 'react-instantsearch';
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
+
+const { searchClient } = instantMeiliSearch(
+  'https://ms-adf78ae33284-106.lon.meilisearch.io',
+  'a63da4928426f12639e19d62886f621130f3fa9ff3c7534c5d179f0f51c4f303'
+);
+
+const App = () => (
+  <InstantSearch
+    indexName="steam-video-games"
+    searchClient={searchClient}
+  >
+  </InstantSearch>
+);
+
+export default App
+```
+
+## 5. Add a search bar and list search results
+
+Add the `SearchBox` and `InfiniteHits` components inside the `InstantSearch` wrapper component. The Hits component accepts a custom Hit component via the `hitComponent` prop, which allows customizing how each search result is rendered.
+Import the CSS library to style the search components.
+
+```jsx
+import React from 'react';
+import { InstantSearch, SearchBox, InfiniteHits } from 'react-instantsearch';
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
+import 'instantsearch.css/themes/satellite.css';
+
+const { searchClient } = instantMeiliSearch(
+  'https://ms-adf78ae33284-106.lon.meilisearch.io',
+  'a63da4928426f12639e19d62886f621130f3fa9ff3c7534c5d179f0f51c4f303'
+);
+
+const App = () => (
+  <InstantSearch
+    indexName="steam-video-games"
+    searchClient={searchClient}
+  >
+    <SearchBox />
+    <InfiniteHits hitComponent={Hit} />
+  </InstantSearch>
+);
+
+const Hit = ({ hit }) => (
+  <article key={hit.id}>
+    <img src={hit.image} alt={hit.name} />
+    <h1>{hit.name}</h1>
+    <p>${hit.description}</p>
+  </article>
+);
+export default App
+```
+
+<Capsule intent="tip">
+Use the following CSS classes to add custom styles to your components:
+`.ais-InstantSearch`, `.ais-SearchBox`, `.ais-InfiniteHits-list`, `.ais-InfiniteHits-item`
+</Capsule>
+
+## 6. Start the app and search as you type
+
+Start the app, open your browser, go to your React app URL, and start searching!
```

Comment:
can you split this? eg.

"start the app by running:
npm run dev
now open your browser and navigate to you React app URL (e.g. localhost:3000), and start searching"